src/index.html

```diff
@@ -194,8 +215,10 @@
 </head>
 <body>
     <div class="error-msg"></div>
+    <div class="msg msg-hidden"></div>
     <img class="logo" src="2_augmint_alternate-logo_reversed-360.jpg" alt="Augmint logo">
     <div class="buttons">
+        <button class="btn store-btn ">Store</button>
```

Comment:
could we rename it to "Save/load params" and move it to the right (after show log button).